fio/func.sh

```
@@ -1,30 +1,39 @@
 }
 
-parse_bw() {
-        local OUTPUT_BW=$1
-        RAND_READ_BW=`cat $OUTPUT_BW | jq '.jobs[0].read.bw_mean'| cut -f1 -d.`
-        RAND_WRITE_BW=`cat $OUTPUT_BW | jq '.jobs[1].write.bw_mean'| cut -f1 -d.`
-        SEQ_READ_BW=`cat $OUTPUT_BW | jq '.jobs[2].read.bw_mean'| cut -f1 -d.`
-        SEQ_WRITE_BW=`cat $OUTPUT_BW | jq '.jobs[3].write.bw_mean'| cut -f1 -d.`
-        CPU_IDLE_PCT_BW=`cat $OUTPUT_BW| jq '.cpu_idleness.system' | cut -f1 -d.`
+parse_write_iops() {
+	local OUTPUT_IOPS=$1
+	RAND_WRITE_IOPS=`cat $OUTPUT_IOPS | jq '.jobs[0].write.iops_mean'| cut -f1 -d.`
```

Shellcheck

[warning] RAND_WRITE_IOPS appears unused. Verify use (or export if used externally).

(SC2034)